### ISSUE TITLE
Platform support for Win32 applications

### ISSUE THREAD

**Issue by longgnc** [PTCInc/vuforia-engine#164]
**Summarize your feature feedback**
I need to run the vuforia packaged AR project on Windows

**Describe your feature**
Just like packaging in Android, you can call the camera and use the camera to recognize the corresponding image or object, my device is the surface pro8

**Use-case description**
Just like packaging in Android, you can call the camera and use the camera to recognize the corresponding image or object, my device is the surface pro8, which is equivalent to being packaged as an exe program and running on Windows

**Business impact**
Successfully help me complete the project requirements

**Workaround**
There is no way, I checked the support device, there is a surface, but I packed the surface, running a black screen, without any prompt

**Additional context**
That's it, I want to package exe, function and mobile phone can be consistent

**Comment by ptc-thesbyro** [comment]
Hi @niefun, 

If I understand correctly, you wish to have build support for Windows other than UWP?

**Comment by longgnc** [comment]
Yes, I have an application running on Microsoft surface tablet (Windows application,.exe suffix), I need to embed the ar made by vuforia into this program and turn it into a module function of this program, may I ask if vuforia has a ready-made solution?

**Comment by ptc-thesbyro** [comment]
Thanks for the clarification. 

We do not currently have a ready-made solution to support Win32 applications. However, I will keep this issue open and bring your support request to the product team. 

The only suggestion I can offer at this moment is to use UWP as the target platform for your AR application. 

There is a possible technical path to run your project as two applications that communicate via a sandbox environment. However, this is challenging, and I would advise against it as it would fall outside our support domain.

**Comment by Altaiar** [comment]
i would like to have it too there is some rare bugs in UWP and its seems deprecated both by microsoft and unity 
vuforia is one of the best AR trackers out there and lack of windows native support is really annoying
